### PR TITLE
Generate logs playbook

### PR DESCRIPTION
**Pull request by saharsabban**
## Status
Ready

## Description
Adding a system playbook for generating log bundle using Demisto rest api integration and send it to logs@demisto.com

## Does it break backward compatibility?
No

## Must have
- [ ] Tests

### CODE REVIEW

File: Playbooks/playbook-Generate_log-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+id: Generate log-bundle
+version: -1
+name: Generate log-bundle
+description: |-
```
Comment (JonathanMeler) [inline Playbooks/playbook-Generate_log-bundle.yml]: Add a proper description

File: Playbooks/playbook-Generate_log-bundle.yml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  "2":
+    id: "2"
+    taskid: fa1c9f48-3f42-4e31-87a9-cfedfe21cf5e
+    type: regular
+    task:
+      id: fa1c9f48-3f42-4e31-87a9-cfedfe21cf5e
+      version: -1
+      name: 'Send logs '
+      description: "Send logs to Demisto"
+      script: '|||send-mail'
```
Comment (JonathanMeler) [inline Playbooks/playbook-Generate_log-bundle.yml]: Add a test that mail-sender integration is enabled.

File: Playbooks/playbook-Generate_log-bundle.yml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      attachIDs:
+        simple: ${lastCompletedTaskEntries}
```
Comment (JonathanMeler) [inline Playbooks/playbook-Generate_log-bundle.yml]: Convert to visual filter

File: Playbooks/playbook-Generate_log-bundle.yml [hunk 4]
```diff
@@ -0,0 +1,139 @@
+id: Generate log-bundle
+version: -1
+name: Generate log-bundle
```
Comment (JonathanMeler) [inline Playbooks/playbook-Generate_log-bundle.yml]: Change name to: Generate Log Bundle - Demisto (verify with @kirbles19 )

Comment (anara123) [reply Playbooks/playbook-Generate_log-bundle.yml]: I would call it `Send Logs to Demisto`

**Comment by saharsabban** [comment]
Will make it as automation.

**Pull request closed without merging**